<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,12">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>LW</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_LW">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_5">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,12">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>RW</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_RW">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,12">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Depth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_depth">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4,12">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_high">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_4">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,4,12">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Center</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_center"/>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,12">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Radius</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_zoom">
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_3">
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_LW</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_RW</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_depth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_high</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_center</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_zoom</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
